projects/packages/jetpack-mu-wpcom/src/features/wpcom-media/wpcom-media-url-upload.php

```diff
@@ -23,10 +23,10 @@ function wpcom_media_url_upload() {
 
 	$data = wp_json_encode(
 		array(
-			'ajaxUrl'  => admin_url( 'admin-ajax.php' ),
-			'action'   => 'wpcom_media_url_upload',
-			'nonce'    => wp_create_nonce( 'wpcom_media_url_upload' ),
-			'isEditor' => $pagenow !== 'upload.php',
+			'ajaxUrl' => admin_url( 'admin-ajax.php' ),
+			'action'  => 'wpcom_media_url_upload',
+			'nonce'   => wp_create_nonce( 'wpcom_media_url_upload' ),
+			'page'    => $pagenow === 'upload.php' ? 'media-library' : 'editor',
```

Comment:
Looks like we need to support `media-new.php` as well. What do you think?

Reply:
How so? The pluploader does not work there...

Reply:
Ok. Let's address this when we want to add the button there

Reply:
I think I remember now; I didn't implement there because the pluploader logic is a bit different there. It's the same reason why VideoPress is only available on `upload.php` but not `media-new.php`